torch/csrc/jit/symbolic_script.cpp

```diff
@@ -1371,6 +1357,29 @@ const std::vector<std::string> functions = {
                 return grad_output, None
             return torch.remainder(self, other), backward
 
+        def AD_column_major(mat):
+            return mat.stride(0)==1 and mat.stride(1) == mat.size(0)
+
+        def AD_mm_backward_self(grad, mat2, mat1):
```

Comment:
Should this be AD_mm_backward_mat1(grad, mat1, mat2)? Currently there's a name mismatch with the addmm implementation and inconsistent argument ordering. For example, AD_mm_backward_self actually computes the gradient of mat1, not self.